core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -200,11 +209,11 @@ public static void set(String key, String value) {
    * @return the value for the given key
    */
   public static String get(String key) {
-    if (!PROPERTIES.containsKey(key)) {
+    if (!PROPERTIES_MAP.containsKey(key)) {
```

Comment:
I agree. Getting directly and dealing with the null are both safer and quicker than the original implementation.